source/cppassist/source/fs/SystemInfo.cpp

```diff
@@ -47,7 +47,11 @@ const std::string & SystemInfo::libExtension()
 
 std::string SystemInfo::homeDir()
 {
-    return std::string(getenv("HOME"));
+    #ifdef WIN32
```

Comment:
I suggest to use our own cmake-init-defined macro `SYSTEM_WINDOWS`.
However, I'm not aware of systems where those two definitions differ.